### DISCUSSION TITLE
Errors in Tweakscale?

### DISCUSSION THREAD

**Discussion by lux7328** [TweakScale/TweakScale#291] [Support] [answered]
So, I am entirely new to add-ons and mods so I really have no idea what I am doing... I downloaded a bunch of mods from curseforge and had a relatively easy time doing that. I started with tweakscale and several parts add-ons. I stayed away from everything that enhanced visuals because well I have a slow laptop. I wanted to add editor extender and hangar extender and once I did everything went downhill, I followed the "Houston we have a problem" instruction as best I could and now finally, I can get KSP to open without Houston having a problem. But when KSP is loading it calls for 2 errors in Tweakscale. I think it's probably something I did wrong. Tweakscale appears to be working the way it's supposed to be so I don't know if this is actually a problem now or if it will be a problem later.

Also I don't know how to pull the "KSP.log"

@Lisias

<img width="515" alt="Screenshot 2023-02-08 000719" src="https://user-images.githubusercontent.com/124810519/217610139-cce0b5bb-5c1d-48dd-9b28-d0f6410ece9c.png">

**Comment by Lisias** [comment, marked as answer]
Hi, @lux7328 !

These are not errors from TweakScale - are errors on patching TweakScale. There're errors on the Module Manager patches used by "HYPERMOD 2.5 20k Special" - that I don't have the slighest idea about, by the way.

You need to reach the maintainer for "HYPERMOD 2.5 20k Special" and file a bug report!

Cheers!

**Reply by Lisias** [reply]
@lux7328 , about `KSP.log` - you will find this find on the same directory where you KSP executable is. All Launchers (Steam, GOG and Epic) allows you to open the directory where the game is, and Windows have an option to show the "target file" by clicking with the Right Button on the Shortcut button.
